src/browser/html/document.zig

```diff
@@ -207,6 +207,23 @@ pub const HTMLDocument = struct {
     pub fn set_bgColor(_: *parser.DocumentHTML, _: []const u8) []const u8 {
         return "";
     }
+
+    pub fn _elementFromPoint(_: *parser.DocumentHTML, x: f32, y: f32, state: *SessionState) !?*parser.Element {
```

Comment:
```suggestion
    pub fn _elementFromPoint(_: *parser.DocumentHTML, x: f32, y: f32, state: *SessionState) !?ElementUnion {
```

Should `ElementUnion` be returned?

Reply:
Yes, probably. If you return it as a `*parser.Element`, it'll be the v8 ObjectTemplate associated with that type, i.e. the `Element` struct in `dom/element.zig`. That's a problem for any "subtype", since you wouldn't be able to call any of the subtype methods. For example, if the element is actually a `HTMLAnchorElement`, you'd want to be able to call `el.href` in JavaScript. It's only by returning the union that the js runtime can pick the correct ObjectTemplate.

Reply:
Are you sure? Happy to change it of course, but the tests do indicate a `[object HTMLDivElement]` is returned.

Reply:
I know that, right now,` elementFromPoint` can only returned a "known" elements, but I'm not sure that's the right behavior, as I pointed out in my comment that I'm not sure if `.{ "document.elementFromPoint(0.5, 0.5)", "null" }` is correct.

Reply:
ElementUnion has been changed, resolving this discussion other discussion continues here: https://github.com/lightpanda-io/browser/pull/642#discussion_r2095245929